README.rst

```diff
@@ -1,7 +1,7 @@
 Thoth Adviser
 -------------
 
-A recommendation engine for project `Thoth <https://github.com/thoth-station/>`_.
+A recommendation engine and software stack generation for project `Thoth <https://github.com/thoth-station/>`_.
```

Comment:
"generator" ?

Reply:
Dependency Monkey? I'm bad at phrasing, so feel free to suggest...

Reply:
then... just replace generaton by generator